LondonBritishLibrary/orient/BLorient787.xml

```diff
@@ -0,0 +1,288 @@
+                                      <measure unit="leaf">164</measure>
+                                      <measure unit="blank">1</measure>
+                                      <dimensions type="outer" unit="in">
+                                          <height>9.5</height>
+                                          <width>8.375</width>
+                                      </dimensions>
+                                  </extent>
+                                </supportDesc>
+
+                              <layoutDesc>
+                                  <layout columns="2" writtenLines="21 23"/>
+                              </layoutDesc>
+                          </objectDesc>
+                          <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                            <desc>Written in a good clear hand; apparently of the <date>17th Cent.</date></desc>
+                            </handNote>
+                            </handDesc>
+
+                            <additions>
+                            <list>
+                            <item xml:id="a1">
+                            <locus from="1v"/>
+                            <desc type="GuestText"><title type="complete" ref="LIT3332Melchiz">A section on Melchizedek</title></desc>
+                            <q xml:lang="gez">ይነግር፡ ብእሲ፡ ልሂቅ፡ ወጠቢብ፡ ለሕገ፡ እግዚአብሔር፡ በእንተ፡ መልከ፡ ጼዴቅ፡</q>
+                            </item>
+
+                            <item xml:id="a2">
+                            <locus from="2r"/>
+                            <desc type="GuestText"><title type="complete" ref="LIT3333Treatise">On the 318 Nicene Fathers with Creed and Anathemas</title></desc>
+                            <q xml:lang="gez"><seg part="I">ነአምን፡ እንከ፡ ህልወ፡ አነ፡ እንዝእ፡ ቀሲስ፡ በቅዳሚ፡ እግዚአብሔር፡ አብ፡ ዘሀሎ፡ አልቦ፡ አመ፡
+                            ኢሀሎ፡ <gap reason="omitted" resp="PRS10171Wright"/></seg></q>
+                            <q xml:lang="gez"><seg part="F">
+                            <locus target="#3v"></locus>
+                            ተፈጸመ፡ ጽሕፈተ፡ ድርሳን፡ ዘቅዱስ፡ ቄርሎስ፡ በረድኤተ፡ እግዚአብሔር፡ አሜን።</seg>
+                          </q>
+                            </item>
+                          </list>
+                        </additions>
+
+                            <bindingDesc>
+                                <binding contemporary="true" xml:id="binding">
+                                    <decoNote xml:id="b1">In the original boards, covered with stamped leather.</decoNote>
+                                    <decoNote xml:id="b2" type="bindingMaterial">
+                                        <material key="wood"/>
+                                        <material key="leather"/>
+
+                                    </decoNote>
+                                </binding>
+                            </bindingDesc>
+
+
+                          </physDesc>
+                          <history>
```

Comment:
You could add information on the date of the origin, as the ms is dated on the basis of plaeography